app/views/support/locations/_form.html.erb

```diff
@@ -1,11 +1,10 @@
-<%= form_with model: [:support, site], local: true, url: form_url, class: "location-form" do |f| %>
-
+<%= form_with model: @location_form, local: true, url: form_url, class: "location-form" do |f| %>
   <%= f.govuk_error_summary %>
 
   <%= f.govuk_text_field :location_name, label: { text: "Name", size: "m" } %>
```

Comment:
Looking at the prototype, i think all of these labels now need to be a size `s`. Eg:

```
<%= f.govuk_text_field :location_name, label: { text: "Name", size: "s" } %>
```

Prototype:

<img width="831" alt="Screenshot 2023-03-22 at 09 49 53" src="https://user-images.githubusercontent.com/616080/226865596-d3fdb46a-78ae-44a4-85c4-8002a1a10cc5.png">

The build:

<img width="765" alt="Screenshot 2023-03-22 at 09 51 03" src="https://user-images.githubusercontent.com/616080/226865901-5597c90e-dee4-4f08-9c95-103259d49a74.png">

Reply:
Done

Reply:
Just looking at the new review app and it's still not quite there. 

- URN hint text needs to be below the input
- Create button needs to say `Continue`
- The address labels are also bold on the prototype?